plugins/org.eclipse.papyrus.uml.diagram.sequence.runtime/src/org/eclipse/papyrus/uml/diagram/sequence/runtime/internal/edit/policies/AbstractSequenceGraphicalNodeEditPolicy.java

```diff
@@ -238,16 +238,16 @@ public Command caseCreateConnectionViewRequest(CreateConnectionViewRequest reque
 
 					AnchorDescriptor anchorDesc = computeAnchoring(location);
 
-					if (MessageUtil.isSynchronous(start.sort) && selfMessage && shouldCreateExecution()) {
```

Comment:
Yeah:  the `isSynchronous` API literally means is the message instantaneous, such that it is constrained to a horizontal visualization.  Sorry if that wasn't clear!